### ISSUE TITLE
Performance Issue with 100 rows and 50 tasks per row

### ISSUE THREAD

**Issue by gouthamsworld** [angular-gantt/angular-gantt#597]
Did anyone tried either one way binding or loading data on api.scroll.on.scroll for better performance. I am loading 100 rows with 6 day date range with 50 tasks in each row and scrolling is pretty bad. Can some one help ?

Thanks
Gouthamn

**Comment by Toilal** [comment]
Would a read only mode fit your need ? It's only an idea for now, but I don't think we can enhance performance now while keeping the two-way data binding.

**Comment by gouthamsworld** [comment]
Yes a full one way binding will be perfect for my need.Is there any other ways of increasing the performance ?


Thanks
Goutham

**Comment by Toilal** [comment]
Rewriting the whole thing using angular2 or React :)

But maybe we could implement something like this in angular-gantt : http://codetunes.com/2015/story-of-angular-watchers-toggler-directive/

**Comment by lfreneda** [comment]
@Toilal, did you found an alternative way to work around the performance issue? 

Rewriting the whole thing right now for me is no a valid option :(

**Comment by lfreneda** [comment]
@gouthamsworld help me out? :)
